packages/grafana-ui/src/components/Forms/Checkbox.tsx

```diff
@@ -59,7 +59,7 @@ export const getCheckboxStyles = stylesFactory((theme: GrafanaTheme2) => {
 
   return {
     wrapper: css`
-      display: flex;
+      display: inline-flex;
```

Comment:
Has this been tested across the entire app and possibly plugins to confirm it doesn't cause layout issues elsewhere?

Reply:
I have just checked it with some plugins and it actually fixes the alignment in some cases too (note the checkbox at the bottom):

| Before | After |
| --- | --- |
| <img width="380" alt="Screenshot 2023-02-01 at 14 10 23" src="https://user-images.githubusercontent.com/100691367/216066240-6ccf7cdb-7a76-4dca-940e-83b57b05a028.png"> | <img width="272" alt="Screenshot 2023-02-01 at 14 10 41" src="https://user-images.githubusercontent.com/100691367/216066187-d8e06efb-94e4-4a1f-9dfb-44ea9f5d2ab3.png"> |

Reply:
So I do think this fixes things to get closer to what they looked like before the PR that introduced the `flex`

Reply:
Ah cool! Thanks for taking the time to double check this @JoaoSilvaGrafana . 👏